src/community/leiden.c

```diff
@@ -1263,3 +1271,229 @@ igraph_error_t igraph_community_leiden(
 
     return IGRAPH_SUCCESS;
 }
+
+/**
+ * \function igraph_community_leiden_simple
+ * \brief Finding community structure using the Leiden algorithm, simple interface.
+ *
+ * This is a simplified interface to \ref igraph_community_leiden() for
+ * convenience purposes. Instead of requiring vertex weights, it allows
+ * choosing from a set of objective functions to maximize, and calculates
+ * the vertex weights and resolution parameter to pass to
```

Comment:
Is it calculating the resolution parameter automatically? The function seems to have a resolution parameter that needs to be specified explicitly.